Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -783,6 +773,35 @@ - (void)bounceMarkdownPreview
     }];
 }
 
+- (void)refreshNavigationBarButtons {
+    
+    self.checklistButton.hidden = !self.isEditing;
```

Comment:
Checked with Sly on this one, will get back to you as soon as I hear from him. We'll _probably_ want to disable, rather than hide.

Reply:
For sure. Let me know and I'll make the change!

Reply:
Replied by Sly below, good to keep. Thank you!!